spec/models/application_spec.rb

```diff
@@ -199,6 +199,18 @@
         is_expected.to match_array([application_1, application_2])
       end
     end
+
+    describe '.waiting_for_evidence', focus: true do
+      let!(:application1) { create :application }
+      let!(:application2) { create :application }
+      let!(:evidence_check) { create :evidence_check, application: application1 }
```

Comment:
Can we use mocks/stubs here?

As it's a model and this is meant to be a unit test.